ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/CommandTopicBackupImpl.java

```diff
@@ -138,13 +138,14 @@ private boolean isRecordInLatestReplay(final ConsumerRecord<byte[], byte[]> reco
   public void writeRecord(final ConsumerRecord<byte[], byte[]> record) {
     if (corruptionDetected) {
       throw new CommandTopicCorruptionException(
-          "Failed to write record due to out of sync command topic and backup file: " + record);
+          "Failed to write record due to out of sync command topic and backup file. "
+              + String.format("partition=%d, offset=%d", record.partition(), record.offset()));
     }
 
     if (record.key() == null || record.value() == null) {
       LOG.warn(String.format("Can't backup a command topic record with a null key/value:"
-              + " key=%s, value=%s, partition=%d, offset=%d",
```

Comment:
We shouldn't log the key and value directly. Operators will still be able to debug, since they can use the partition and offset to fetch the record from the topic.